### ISSUE TITLE
Cannot open Aurous in Ubuntu14.04 x64

### ISSUE THREAD

**Issue by Mikebarson** [AurousGroup/Bug-Tracker#146]
I can't open Aurous in Ubuntu 14.04 x64. I've installed all the java requirements and even the extra extensions needed. After running the Java- jar Aurous.jar it says launching but nothing opens.

**Comment by mauriciomarinho** [comment]
Same for me, here is the error code:

Update finished
Check finished
Launching aurous...
UTF-8
Exception in Application start method
out 14, 2015 3:22:20 AM me.aurous.launchwrapper.Updater main
GRAVE: null
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at me.aurous.launchwrapper.AurousLoader.launchAurous(AurousLoader.java:39)
	at me.aurous.launchwrapper.Updater.main(Updater.java:32)
Caused by: java.lang.RuntimeException: Exception in Application start method
	at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:917)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$156(LauncherImpl.java:182)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.teamdev.jxbrowser.chromium.internal.ipc.IPCException: Failed to start IPC process.
	at com.teamdev.jxbrowser.chromium.internal.ipc.d.run(Unknown Source)
	... 1 more
Caused by: java.lang.RuntimeException: java.io.IOException: Cannot run program "/home/mauricio/App/aurous/linux/jxbrowser-chromium" (in directory "/home/mauricio/App/aurous/linux"): error=13, Permissão negada
	at com.teamdev.jxbrowser.chromium.internal.ipc.ChromiumProcess.start(Unknown Source)
	... 2 more
Caused by: java.io.IOException: Cannot run program "/home/mauricio/App/aurous/linux/jxbrowser-chromium" (in directory "/home/mauricio/App/aurous/linux"): error=13, Permissão negada
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	... 3 more
Caused by: java.io.IOException: error=13, Permissão negada
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:248)
	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 3 more

Exception in thread "Thread-4" java.lang.NullPointerException
	at com.teamdev.jxbrowser.chromium.internal.ipc.IPC.c(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.ipc.IPC.b(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.ipc.IPC.a(Unknown Source)
	at com.teamdev.jxbrowser.chromium.internal.ipc.b.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)

**Comment by mauriciomarinho** [comment]
Giving executable permissions to "aurous/linux/jxbrowser-chromium" seems to have fixed it.

**Comment by sevenskull** [comment]
I use Linux Mint 17.2 (ubuntu 14.04) but even make jxbrowser executable didn't work.

**Comment by Revod** [comment]
I Have elementary OS freya (ubuntu 14.04) Aurous work but in the terminal get:

'''
Update finished
Check finished
Disposing updater... 
Launching aurous...
Prism-ES2 Error : GL_VERSION (major.minor) = 1.4

(java:4578): Gtk-WARNING **: Imposible encontrar el motor de temas en la ruta al _modulo: «adwaita»,

(java:4578): Gtk-WARNING **: Imposible encontrar el motor de temas en la ruta al _modulo: «adwaita»,
UTF-8
http://localhost:9222/devtools/devtools.html?ws=localhost:9222/devtools/page/76CB7A7F-7E2D-469A-AB83-7AEAA234F52A

'''

**Comment by mazulo** [comment]
The same problem here. When I run `java -jar Aurous.jar` I get:

````shell
Update finished
Check finished
Launching aurous...
UTF-8
````

But the Aurous didn't launch :(

**Comment by dz0ny** [comment]
@mazulo  see http://askubuntu.com/a/369311

tldr;
jxbrowser-chromium is way too old for "modern" linux system
¨¨
linux/jxbrowser-chromium 
../../third_party/tcmalloc/chromium/src/free_list.h:118] Memory corruption detected. 
fish: 'linux/jxbrowser-chromium' terminated by signal SIGSEGV (Address boundary error)
¨¨

**Comment by maldad** [comment]
Did somedoby have problems with Gtk? running ``` java -jar Aurous.jar ```
gives me this:
```
Update finished
Check finished
Launching aurous...
Gtk-Message: Failed to load module "pantheon-filechooser-module"
Gtk-Message: Failed to load module "canberra-gtk-module"
UTF-8
```

**Comment by Revod** [comment]
@maldad  I a little similar to that: https://github.com/AurousGroup/Bug-Tracker/issues/146#issuecomment-148077859

and I think this answer could help you. http://elementaryos.stackexchange.com/questions/333/gtk-error-while-running-a-pyplot-application/339#339

**Comment by maldad** [comment]
@Revod thanks, i tried updating my system, i have elementary 0.3.1, but still the same problem, well, this works...
http://elementaryos.stackexchange.com/questions/1859/gtk-message-failed-to-load-module-pantheon-filechooser-module

##### Attention if somebody gonna  try this, use ``` reset ``` instead of set, like this:
```
gsettings reset org.gnome.settings-daemon.plugins.xsettings enabled-gtk-modules
```

Also I'm still getting canberra issues:
```
maldad@CthulhuFyare ~/P/Aurous> sudo java -jar Aurous.jar 
[sudo] password for maldad: 
Update finished
Check finished
Launching aurous...
Gtk-Message: Failed to load module "canberra-gtk-module"
UTF-8
http://localhost:9222/devtools/devtools.html?ws=localhost:9222/devtools/page/385D1B41-0E56-4821-90BB-B4E7ADA7659A
```

But Aurus is running, although it freezes most times, idk, gonna try in Windows later...

Saludos!!

**Comment by Revod** [comment]
For the canberra-gtk-module try with this:
http://askubuntu.com/a/349672/452466

I open the app, add the "musica" directory to the app and the app crash and get:
```java
java.nio.file.NoSuchFileException: /home/revod/MÃºsica
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:144)
	at sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
	at java.nio.file.Files.readAttributes(Files.java:1734)
	at java.nio.file.FileTreeWalker.getAttributes(FileTreeWalker.java:219)
	at java.nio.file.FileTreeWalker.visit(FileTreeWalker.java:276)
	at java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:322)
	at java.nio.file.Files.walkFileTree(Files.java:2659)
	at java.nio.file.Files.walkFileTree(Files.java:2739)
	at me.aurous.local.media.MediaScanner.walkFolders(MediaScanner.java:53)
	at me.aurous.js.AurousBridge$19.run(AurousBridge.java:69)
```

**Comment by Mikebarson** [comment]
I've read the others and they say to sudo java -jar Aurous.jar....and it worked for me.

**Comment by mazulo** [comment]
@Mikebarson I used `sudo` and it worked. But it's so slow

**Comment by Mikebarson** [comment]
Yeah It is quite slow...

**Comment by Revod** [comment]
Yeah, need put in the terminal with sudo and the app is running but very slow and in the terminal I still getting error's

**Comment by Mikebarson** [comment]
What OS are you operating on?

**Comment by Revod** [comment]
Elementary OS Freya (based ubuntu 14.04) 32 bit.
Get
```java
Prism-ES2 Error : GL_VERSION (major.minor) = 1.4

(java:2636): Gtk-WARNING **: Imposible encontrar el motor de temas en la ruta al _modulo: «adwaita»,

(java:2636): Gtk-WARNING **: Imposible encontrar el motor de temas en la ruta al _modulo: «adwaita»,
UTF-8
```

and 
```java
xception in thread "JavaFX Application Thread" java.lang.NoClassDefFoundError: javafx/scene/control/Alert
	at me.aurous.exceptions.ExceptionWidget$1.run(ExceptionWidget.java:28)
	at com.sun.javafx.application.PlatformImpl$6$1.run(PlatformImpl.java:297)
	at com.sun.javafx.application.PlatformImpl$6$1.run(PlatformImpl.java:294)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl$6.run(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at com.sun.glass.ui.gtk.GtkApplication.access$200(GtkApplication.java:47)
	at com.sun.glass.ui.gtk.GtkApplication$5$1.run(GtkApplication.java:137)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.lang.ClassNotFoundException: javafx.scene.control.Alert
	at java.net.URLClassLoader$1.run(URLClassLoader.java:359)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:348)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:347)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:423)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:356)
	... 10 more
java.nio.file.NoSuchFileException: /home/revod/MÃºsica
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:144)
	at sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
	at java.nio.file.Files.readAttributes(Files.java:1734)
	at java.nio.file.FileTreeWalker.getAttributes(FileTreeWalker.java:219)
	at java.nio.file.FileTreeWalker.visit(FileTreeWalker.java:276)
	at java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:322)
	at java.nio.file.Files.walkFileTree(Files.java:2659)
	at java.nio.file.Files.walkFileTree(Files.java:2739)
	at me.aurous.local.media.MediaScanner.walkFolders(MediaScanner.java:53)
	at me.aurous.js.AurousBridge$19.run(AurousBridge.java:69)
```

**Comment by mazulo** [comment]
My OS: Ubuntu 14.04
